### PR TITLE
add support for personal access tokens

### PR DESCRIPTION
**Pull request by noonhub**

### CODE REVIEW

File: src/main/java/com/asana/Client.java [hunk 1]
```diff
@@ -20,8 +21,7 @@
 /**
  * Main Asana client class
  */
-public class Client
-{
+public class Client {
```
Comment (epelz) [inline src/main/java/com/asana/Client.java]: Lots of style changes to this file that seem unrelated, does this make it consistent with the rest of the repo?

**Comment by epelz** [comment]
Since we don't have unit tests for this, I assume you tested everything pretty thoroughly? If so, looks good and I can merge.

File: src/main/java/com/asana/Client.java (hunk 1 above)
Comment (noonhub) [inline src/main/java/com/asana/Client.java]: The repo is headed in this direction in terms of style. Will make a sweeping change in a different pull-request.

**Comment by noonhub** [comment]
Yeah, tested thoroughly :)
